documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -0,0 +1,394 @@
+// This module is included in the following files:
+//
+// master.adoc
+
+[id='con-common-configuration-properties-{context}']
+== Common configuration properties
+
+Common configuration properties apply to more than one resource.
+
+[id='con-common-configuration-replicas-{context}']
+.`replicas`
+
+Use the `replicas` property to configure replicas.
+
+The type of replication depends on the resource.
+
+* `KafkaTopic` uses a replication factor to configure the number of replicas of each partition within a Kafka cluster.
+* Kafka components use replicas to configure the number of pods in a deployment to provide better availability and scalability.
+
+*MirrorMaker*
+
+When running Kafka MirrorMaker on Kubernetes it is not absolutely necessary to run multiple replicas of the Kafka MirrorMaker for high availability.
+When the node where the Kafka MirrorMaker has deployed crashes, Kubernetes will automatically reschedule the Kafka MirrorMaker pod to a different node.
+However, running Kafka MirrorMaker with multiple replicas can provide faster failover times as the other nodes will be up and running.
+
+[id='con-common-configuration-bootstrap-{context}']
+.`bootstrapServers`
+
+Use the `bootstrapServers` property to configure a list of bootstrap servers.
+
+The bootstrap server lists can refer to Kafka clusters that are not deployed in the same Kubernetes cluster.
+They can also refer to a Kafka cluster not deployed by Strimzi.
+
+If on the same Kubernetes cluster, each list must ideally contain the Kafka cluster bootstrap service which is named `_CLUSTER-NAME_-kafka-bootstrap` and a port of 9092 for plain traffic or 9093 for encrypted traffic.
+If deployed by Strimzi but on different Kubernetes clusters, the list content depends on the approach used for exposing the clusters (routes, nodeports or loadbalancers).
+
+When using Kafka with a Kafka cluster not managed by Strimzi, you can specify the bootstrap servers list according to the configuration of the given cluster.
+
+[id='con-common-configuration-cipher-{context}']
+.`SSL` cipher suites
+
+Use the three allowed `ssl` configuration options to run external listeners with a specific _cipher suite_ for a TLS version.
+A _cipher suite_ combines algorithms for secure connection and data transfer.
+
+.Example SSL configuration
+[source,yaml,subs="attributes+"]
+----
+spec:
+  producer:
+    config:
+      ssl.cipher.suites: "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384" <1>
+      ssl.enabled.protocols: "TLSv1.2" <2>
+      ssl.protocol: "TLSv1.2" <3>
```

Comment:
Thsi shows up in many different variants ... I wonder if it should be just

```suggestion
# ...
config:
  ssl.cipher.suites: "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384" <1>
  ssl.enabled.protocols: "TLSv1.2" <2>
  ssl.protocol: "TLSv1.2" <3>
# ...
```

in the example.